functions.py

```diff
@@ -0,0 +1,63 @@
+import time
+from objects import *
+
+
+# score_a = 0
+# score_b = 0
+
+# rockets movement
+
+def rocket_left_up():
+    y = rocket_left.ycor()
+    if y < 250:
+        y += 20
+        rocket_left.sety(y)
+
+
+def rocket_left_down():
+    y = rocket_left.ycor()
+    if y > -250:
+        y -= 20
+        rocket_left.sety(y)
+
+
+def rocket_right_up():
+    x = rocket_right.ycor()
+    if x < 250:
+        x += 20
+        rocket_right.sety(x)
+
+
+def rocket_right_down():
+    x = rocket_right.ycor()
+    if x > - 250:
+        x -= 20
+        rocket_right.sety(x)
```

Comment:
Looks like the rocket is better to be organised as a class. It will look something like:

```python

class Rocket:
    def __init__(self, position_x, position_y):
        # rocket object creation should happen here
        pass

    def up(self):
        # implement rocket movement inside this method
        pass

    def down(self):
        pass
    
    # implement other methods on the rocket within this class
```